index.bs

```diff
@@ -2029,12 +2033,12 @@ an |ownership type|, a |serialization internal map|, a |realm| and a |session|:
     <dt>|value| is a [=platform object=] that implements {{NodeList}}
     <dd>Let |remote value| be [=serialize an Array-like=] with
         <code>NodeListRemoteValue</code>,|handle id|, |known object|, |value|,
-        |max depth|, |ownership type|, |serialization internal map|, and |realm|.
+        |serialization options|, |ownership type|, |serialization internal map|, and |realm|.
```

Comment:
Shouldn't we distinguish between `NodeList`+`HTMLCollection` and `Array`, and use  `maxDomDepth` / `maxObjectDepth` accordingly?

Reply:
The fact that `HTMLCollection` and so on are platform objects rather than JS runtime objects seems less important than the fact that they're not part of the DOM.